src/doc/unstable-book/src/library-features/try-trait.md

```diff
@@ -0,0 +1,7 @@
+# `try_trait`
+
+The tracking issue for this feature is: [#31436]
```

Comment:
Added a bit more unstable-book text.  I'm uncertain exactly what's supposed to be in it; is there anything in particular you'd like me to address there?